.github/workflows/deploy.yml

```diff
@@ -0,0 +1,18 @@
+name: Deploy
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: ["Build"]
```

Comment:
I think the workflows: definition needs to change to the name of the build workflow, which I believe is ["Build & Push to ECR" here](https://github.com/department-of-veterans-affairs/vets-api/pull/16582/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R1)